pkg/docs/sql_schema.md

```diff
@@ -95,4 +97,78 @@ Example query to look at all the series:
 -----------+---------+------------+-------
          4 | {3,4}   | dev        | pinky
          5 | {5,6}   | production | brain
-```
\ No newline at end of file
+```
+
+## Series Selectors
+
+We have added simple-to-use series selectors for filtering series in either of the two views above.
+
+### Containment
+
+To test whether a series contains a json fragment you can use the `@>` containment operator.
+
+For example to find all metrics on in the dev namespace and on node pinky, run:
+
+```SQL
+SELECT *
+FROM cpu_usage  u
+WHERE labels @> jsonb '{"namespace":"dev", "node": "pinky"'}
+```
+
+### Label Matchers
+
+You can also match series using label matchers to create predicates on values of particular
+label keys. This is very similar to the label matchers available in PromQL.
+
+For example to find all metrics on in the dev namespace and on node brain using
+label matcher, you can run:
+
+```SQL
+SELECT *
+FROM cpu_usage  u
+WHERE labels ? ('namepace' == 'dev') AND labels ? ('node' == 'brain')
+```
+
+Label matchers are formed by using a qualifier of the form `labels ? (<tag_key> <operator> <pattern>)`.
+There are four operators,
+
+- `==` match tag values that are equal to the pattern
+- `!==` match tag value that are not equal to the pattern
+- `==~` match tag values that match the pattern as a regex
+- `!=~` match tag values that are not equal to the pattern
+
+These four matchers correspond to each of the four selectors in PromQL but with slightly
+different names (to avoid clashing with other PostgreSQL operators). They can
+be combined together using any boolean logic with any arbitrary where clauses.
+
+For those coming from PromQL there are a few differences to keep in mind:
+- Regexes are not anchored for you. Although, you can of course add anchors (`^$`) yourself.
+- The logic for whether series that are missing the tag key pass the qualifier is slightly different:
+  The `!==` and `!=~` do always match series with missing keys while `==` and `==~` do not.
+
+
+### Equivalence
+
+The `eq` function tests equivalence between labels without comparing the metric name (`__name__`) label key.
+Thus, it can be used to compare across metrics or within a metric.
+
+For example, to join 2 series that are scraped at the same time:
+
+```SQL
+SELECT *
+FROM cpu_usage  u
+INNER JOIN cpu_total t  ON (u.time=t.time AND eq(u.labels, t.labels))
+```
+
+You can also use eq to compare to a json labels object:
+
+```SQL
+SELECT *
+FROM cpu_usage
+WHERE eq(labels , jsonb '{“namespace”:”prod”,”node”:”pinky”,"zone":"us-a1-east"}')
+```
+
+Note the above json equivalence requires the entire json object
+to be specified. If you want a partial match see the Containment
+section above.
```

Comment:
I think we should cut this, or expand it to be clear that exactness is required for `eq(label_ids, label_ids)` as well